### PR TITLE
Feature/dw ca avro schema generation

### PR DESCRIPTION
**Pull request by aalbatross**
1.A project to generate DwCA related terms in avro
2. putting the generated schema in core project.

### CODE REVIEW

File: DwCASchemaGenerator/pom.xml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        <configuration>
+          <source>1.8</source>
+          <target>1.8</target>
+        </configuration>
+      </plugin>
+    </plugins>
+    <!-- </pluginManagement> -->
+  </build>
+
+  <dependencies>
```
Comment (fmendezh) [inline DwCASchemaGenerator/pom.xml]: all dependencies versions must be moved to a properties section for better management and keep consistency with other gbif projects

File: DwCASchemaGenerator/pom.xml [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+      <artifactId>avro</artifactId>
+      <version>1.8.2</version>
+    </dependency>
+    <dependency>
+      <groupId>org.jsoup</groupId>
+      <artifactId>jsoup</artifactId>
+      <version>1.8.3</version>
+    </dependency>
+    <dependency>
+      <groupId>org.testng</groupId>
```
Comment (fmendezh) [inline DwCASchemaGenerator/pom.xml]: better if we stick to use junit by now

**Comment by cgendreau** [comment]
Just a quick note and please remember I don't have all the information.

My first understanding was that this was not necessary (see https://github.com/gbif/pipelines/issues/13). If that changed please let me (us) know.

If it is now required (which I assume it is) and it's there to stay, I would suggest to parse to the XML instead of the HTML page. @mdoering would now better what is the better source to use but I know a XML is available on GitHub (https://github.com/tdwg/dwc/blob/master/standard/documents/xml/tdwg_dwcterms.xsd).

**Comment by timrobertson100** [comment]
XML versions are on http://rs.gbif.org/core/

File: DwCASchemaGenerator/src/main/java/org/gbif/DwCAAvroSchemaGenerator.java [hunk 3]
```diff
@@ -0,0 +1,48 @@
+package org.gbif;
```
Comment (fmendezh) [inline DwCASchemaGenerator/src/main/java/org/gbif/DwCAAvroSchemaGenerator.java]: This project doesn't follow the name convention for Maven artifacts, a better approach is to generate  Avro schemas form XSD or XML files, there are several libraries for that, take for example: https://github.com/elodina/xml-avro

File: DwCASchemaGenerator/src/main/java/org/gbif/dwca/generator/util/SchemaType.java [hunk 4]
```diff
@@ -0,0 +1,10 @@
+package org.gbif.dwca.generator.util;
+
+/**
+ * Type of schema
+ *
+ * @author clf358
```
Comment (fmendezh) [inline DwCASchemaGenerator/src/main/java/org/gbif/dwca/generator/util/SchemaType.java]: we discourage the use of @autor

**Pull request closed without merging**